test/legacy/from-dep-tree.test.ts

```diff
@@ -260,6 +260,46 @@ describe('depTreeToGraph with funky pipes in the version', () => {
   });
 });
 
+describe('depTreeToGraph with purl', () => {
+  const depTree = {
+    name: 'oak',
+    version: '1.0',
+    purl: 'pkg:deb/oak@1.0',
+    dependencies: {
+      foo: {
+        version: '2|3',
+        purl: 'pkg:deb/foo@2|3', // i don't think that this is valid purl?
```

Comment:
> i don't think that this is valid purl?

But then why create that as an example? Where did you get the `|` stuff?

Reply:
The above test, called `depTreeToGraph with funky pipes in the version` 😄 

I need to remove that comment though, because it technically is valid purl (and it also works). 

But I'll probably use normal versions instead anyway, unless you see a reason to test a weird version like that :)

Reply:
Yeh, would just use normal versions as the pipe thing isn't what's under test here